up-and-running/auth/custom.md

```diff
@@ -0,0 +1,92 @@
+# Custom Authentication
+
+If authentication system use in your organization is not listed in [npme's supported authentication methods](/up-and-running/auth/#authentication) you can use custom authentication, we have provided following two custom authentication plugins:
```

Comment:
`If authentication system use in your organization` -> `If your authentication system is not listed in`

`you can use custom authentication, we have provided following two custom authentication plugins:` -> `you may be able to an external open-source plugin (or write your own)`. Here are some of the OSS plugins that exist: